lib/web/app/handler.go

```diff
@@ -427,9 +430,81 @@ func (h *Handler) getAppSessionFromCert(r *http.Request) (types.WebSession, erro
 		})
 		return nil, err
 	}
+
+	if identity.AzureIdentities != nil {
+		if err := h.tryResignAzureJWTCookie(r, ws); err != nil {
+			h.log.WithError(err).Debugf("failed to re-sign jwt cookie")
+		}
+	}
```

Comment:
We should verify the request is targeting actual Azure, rather than some other app. I think that checking for azure identities is not sufficient.

Reply:
Ah good catch, I meant to check `identity.RouteToApp.AzureIdentity` here, which IIUC should only be set for Azure app sessions. I can also add a check for the actual app if we think it's needed.